### PR TITLE
Edit blog post

### PR DESCRIPTION
**Pull request by fmerian**

### CODE REVIEW

File: apps/marketing/content/blog/switching-from-slack-to-discord [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Switching to Discord
-description: Join us on documen.so/discord to share ideas, support, and memes.
+description: The Documenso community is growing and we feel the need to have a more community- and developer-friendly environment. We're switching to Discord.
```
Comment (dguyen) [inline apps/marketing/content/blog/switching-from-slack-to-discord:3]: Is the `-` intentional here `community- and`?

File: apps/marketing/content/blog/switching-from-slack-to-discord [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (dguyen) [inline apps/marketing/content/blog/switching-from-slack-to-discord:1]: There are currently some issues with this file which is causing it to not render correctly locally:

1. This file needs the `.mdx` file extension
2. Having tabs before lists is causing the following issue:

```
     • "blog/switching-from-slack-to-discord.mdx" failed with YAMLParseError: Tabs are not allowed as indentation at line 10, column 1:
     
       - Announcement
        - Community
     ^
```

The solution would be to change the tab before `  - Community` on line 10 to two spaces.

**Review by dguyen — CHANGES_REQUESTED** [review]

File: apps/marketing/content/blog/switching-from-slack-to-discord (hunk 2 above)
Comment (dguyen) [reply apps/marketing/content/blog/switching-from-slack-to-discord:1]: You can paste this file into something like this which will help format it:

https://codebeautify.org/mdx-formatter

File: apps/marketing/content/blog/switching-from-slack-to-discord (hunk 1 above)
Comment (fmerian) [reply apps/marketing/content/blog/switching-from-slack-to-discord]: oops! fixed

File: apps/marketing/content/blog/switching-from-slack-to-discord (hunk 2 above)
Comment (fmerian) [reply apps/marketing/content/blog/switching-from-slack-to-discord:1]: my bad! also fixed

**Review by dguyen — APPROVED** [review]
Thanks!

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by fmerian** [comment]
> Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA.

thanks, @Mythie — done!

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by Mythie** [comment]
It looks like you've already accepted our Contributor License Agreement, thanks so much! :tada:

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by Mythie** [comment]
It looks like you've already accepted our Contributor License Agreement, thanks so much! :tada:

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by Mythie** [comment]
It looks like you've already accepted our Contributor License Agreement, thanks so much! :tada:

**Pull request merged**